lib/policies/msRestUserAgentPolicy.browser.ts

```diff
@@ -9,7 +9,7 @@
 import { TelemetryInfo } from "./userAgentPolicy";
 
 interface NavigatorEx extends Navigator {
-  readonly oscpu: string | undefined;
+  readonly oscpu: string;
```

Comment:
@aby040 - How is this change related to the current problem being fixed?

Reply:
Sorry, this was accidentally added to this merge request. This change was made to fix some build errors.